permuta/patterns/patt.py

```diff
@@ -2,6 +2,7 @@
 from typing import TYPE_CHECKING, Iterator, Tuple
 
 if TYPE_CHECKING:
+    # pylint: disable=cyclic-import
```

Comment:
I thought pylint was smart to notice this is not cyclic

Reply:
Without comment
```sh
python -m pylint .\permuta\patterns        
************* Module permuta.patterns.bivincularpatt
permuta\patterns\bivincularpatt.py:81:2: W0511: TODO: Optimize me for Bivincular patterns (fixme)
************* Module permuta.patterns.perm
permuta\patterns\perm.py:1:0: R0401: Cyclic import (permuta.patterns.patt -> permuta.patterns.perm) (cyclic-import)
```